.gitignore

```diff
@@ -1 +1,2 @@
 build_artefacts
+.idea/
```

Comment:
Please break this out as a separate PR. Also please use `[skip ci]` in the commit message for that change. Thanks.